dtm0/it/all2all/README.md

```diff
@@ -92,3 +92,21 @@ Now reload the daemon & stop the hare-consul-agent using following command
 [root@ssc-vm-c-1966 hare]# sudo systemctl daemon-reload && sleep 5 && service hare-consul-agent stop 
 ```
 Try running the test again. Refer [this](https://github.com/Seagate/cortx-motr/blob/dtm0-main/dtm0/it/all2all/README.md#command-to-run-this-test)
+
+## Known Issues
+### 1. all2all integration test performs all tests and at the end shows test status as FAIL because of m0d service failed.
+Example test log is as shown below:
+```sh
+ERR: Process m0d@0x7200000000000001:0xc.service failed
+ERR: Process m0d@0x7200000000000001:0x1a.service failed
+ERR: Process m0d@0x7200000000000001:0x28.service failed
+ERR: TEST STATUS: FAIL
+```
+In above case, in journalctl logs if we see the m0d services getting killed because of timed out like shown below, then it is not an issue with all2all test.
+```sh
+m0d@0x7200000000000001:0x28.service stop-sigterm timed out. Killing
+```
+### 2. If message shown below appears multiple times in journalctl log, then it is not an issue with all2all or dtm0 code.
+```sh
+motr[08732]: 4b30 ERROR [rpc/conn.c:1062:m0_rpc_conn_terminate] <! rc=-125
```

Comment:
What caused this error?
It is harmful?

Reply:
@ivan-alekhin  could you please comment?

Reply:
> could you please comment?

I cannot confirm that "it is not an issue with all2all or dtm0 code". @rajatpatil98, could you please add more details here? did we get a panic? does it happen at the end of the test?

> What caused this error? It is harmful?

@mehjoshi  and @sergey-shilov are working right now on a similar issue  -- m0_panic observed when DTM0 service is about to be stopped. So that, I suspect that this -125 error could cause a panic here too. In this case it is quite harmful, and this entry has to be removed from this list (or at least we need to state that this issue has direct relation to DTM0). Let's wait until Rajat adds more details on that.

Reply:
Removed this issue from list, It is not reproducible so we should not put it into list of known issues.